cmd/tether/utils/hostname_linux.go

```diff
@@ -23,6 +25,8 @@ import (
 	"github.com/vmware/vic/pkg/trace"
 )
 
+var hostnameFile = "/etc/hostname"
```

Comment:
wanted to leave it open for being modified by tests if I could find a neat way of doing so